cmd/lora/main.go

```diff
@@ -115,7 +119,14 @@ func main() {
 		}, []string{"method"}),
 	)
 
-	go subscribeToLoRaBroker(svc, mqttConn, logger)
+	msub, err := mqttsub.NewSubscriber(cfg.loraMsgURL, cfg.subTimeout, logger)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Failed to create MQTT subscriber: %s", err))
```

Comment:
I thought we switched to `errors.Wrap()`? I see `fmt.Sprintf` on a lot of places, not just here.